be/src/exec/pipeline/pipeline_driver.h

```diff
@@ -397,6 +400,10 @@ class PipelineDriver {
     // Whether the query can be expirable or not.
     virtual bool is_query_never_expired() { return false; }
     bool is_epoch_finished() { return _state == DriverState::EPOCH_FINISH; }
+    bool is_epoch_finishing() { return _state == DriverState::EPOCH_PENDING_FINISH; }
+    bool is_still_epoch_finishing() {
```

Comment:
What's the semantic and usage of this interface? Could you please add some comment for it